packages/core/src/browser/status-bar/status-bar.tsx

```diff
@@ -44,20 +44,17 @@ export interface StatusBarEntry {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     arguments?: any[];
     priority?: number;
+    accessibilityInformation?: {
+        label: string,
+        role?: string;
```

Comment:
I think we can probably create an `accessibility.ts` file in `@theia/core/src/common` so it can be re-used (eventually also by `timeline`).